ui/components/Filters.js

```diff
@@ -23,13 +23,14 @@ import Moment from "react-moment";
 import { withSnackbar } from "notistack";
 import CloseIcon from "@material-ui/icons/Close";
 import EditIcon from "@material-ui/icons/Edit";
-import DoneAllIcon from '@material-ui/icons/DoneAll';
+import DoneAllIcon from "@material-ui/icons/DoneAll";
 import { toggleCatalogContent, updateProgress } from "../lib/store";
 import TableSortLabel from "@material-ui/core/TableSortLabel";
 import dataFetch from "../lib/data-fetch";
+import PromptComponent from "./PromptComponent";
 import UploadImport from "./UploadImport";
-import FullscreenIcon from '@material-ui/icons/Fullscreen';
-import FullscreenExitIcon from '@material-ui/icons/FullscreenExit';
+import FullscreenIcon from "@material-ui/icons/Fullscreen";
+import FullscreenExitIcon from "@material-ui/icons/FullscreenExit";
```

Comment:
Are you using some other lint extension to format code? It has formatted multiple line of code.

Reply:
Yes I am using Prettier

Reply:
Gotcha, please don't use it as we are having a recommit linter that will format it. Could you please revert prettier changes as it will affect the code formatting and file edit history?